pkg/specgen/generate/oci.go

```diff
@@ -46,13 +45,13 @@ func makeCommand(s *specgen.SpecGenerator, imageData *libimage.ImageData) ([]str
 	finalCommand = append(finalCommand, command...)
 
 	if len(finalCommand) == 0 {
-		return nil, fmt.Errorf("no command or entrypoint provided, and no CMD or ENTRYPOINT from image")
+		finalCommand = []string{""}
```

Comment:
If we move forward, I'd like to have a `logrus.Debug("no command or entrypoint provided, and no CMD or ENTRYPOINT from image: defaulting to empty string")` or something down these lines.

Reply:
Done.